requirements.txt

```diff
@@ -1,4 +1,4 @@
-ansible>=2.7.8
+ansible>=2.7.8,<2.8
```

Comment:
I'd rather stick to a specific version (2.7.10) so it works similarly for people.

Reply:
Isn't this anyways just mostly used in CI? I hope people are not installing Ansible for the first time when starting to use Kubespray...

Reply:
It is. But I'd rather stick to a specific version in a release, so we make sure it "works" in backward. If we hard pin it to 2.7.10 we are sure people actually USE that version (if they at least do pip install before they do cluster stuff)

Reply:
https://github.com/kubernetes-sigs/kubespray#requirements lists "Ansible version ... (or newer) [...]" - this then should also probably be changed to something like "Ansible version 2.7.10, all older and newer versions are supported on a best effort basis but not guaranteed to work"?

Reply:
Good idea. Let's update that! @ccravens can you do so ?

Reply:
sorry, didn't see this mention, I got the 2.7.10 enforcement pushed up now for review

Reply:
Please rebase from master to make the CI work.

Reply:
@MarkusTeufelberger for our case we are attempting to install k8s using vanilla AWS EC2 AMI that do not have ansible installed, so the ansible dependency is being installed through the requirements.txt file

Reply:
@woopstar I just noticed when rebasing master that it seems they are enforcing ansible to 2.7.8 now in master... Should we leave as 2.7.8? Or updated to 2.7.10?

Reply:
Update to 2.7.8 so it follows master branch or update to 2.7.10 and also do another PR to update to 2.7.10 in master ;)